opm/simulators/flow/KeywordValidation.hpp

```diff
@@ -91,15 +92,21 @@ namespace KeywordValidation
     class KeywordValidator
     {
     public:
-        KeywordValidator(const UnsupportedKeywords& keywords,
-                         const PartiallySupportedKeywords<std::string>& string_items,
-                         const PartiallySupportedKeywords<int>& int_items,
-                         const PartiallySupportedKeywords<double>& double_items,
+        KeywordValidator(const UnsupportedKeywords& unsupported_keywords,
```

Comment:
Imho this is getting out of hand. I'd introduce some struct, e.g.
```c++
struct Supported
{
  SupportedKeywords<std::string>& string;
  SupportedKeywords<int>& int;
  SupportedKeywords<double>& double;
};
```
and adjust constructor to
```c++
KeywordValidator(unsupported, partially_supported, fully_supported, special)
```
and adjust getters accordingly.